### PR TITLE
Update NodeStatus

### PR DESCRIPTION
**Pull request by loony-bean**
+Deleted
-Started
-Available

### CODE REVIEW

**Comment by tstaetter** [comment]
i guess we can omit 
```
enum NodeStatus {
    option allow_alias = true; // <-- Remove me
    …
}
```

the allow_alias now?

File: README.md [hunk 1]
```diff
@@ -63,10 +63,21 @@ Info updates will be sent from BV to API. Possible updates:
...
 
 
+#### Nodes state flow
+
+```mermaid
+graph TD;
+    Creating --> Stopped;
+    Stopped --> Starting --> Running;
+    Running --> Stopping --> Stopped;
+    Running --> Upgrading --> Upgraded --> Running;
```
Comment (tstaetter) [inline README.md:73]: I'd suggest going from "upgraded" to "starting"

Comment (loony-bean) [reply README.md:73]: not sure, it will not not symmetrical.. to start something you first need to stop it

Comment (tstaetter) [reply README.md:73]: doesn't BV need to stop the VM before being able to upgrade?

Comment (loony-bean) [reply README.md:73]: depends on how we implement it i guess
we can do live update, or we can destroy node and create a new one from another image

**Review by tstaetter — APPROVED** [review]

**Pull request merged**